index.html

```diff
@@ -886,9 +886,10 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
   <script type="text/javascript" src="scripts/misc.js"></script>
   <script type="text/javascript" src="scripts/wallet.js"></script>
   <script type="text/javascript" src="scripts/network.js"></script>
-
+  <script type="text/javascript" src="scripts/mempool.js"></script>
   <script>
     // WALLET STATE DATA
+    let mempool = new Mempool();
     let cachedUTXOs = [];
```

Comment:
Remove all references of these two variables, I think @panleone left them for testing purposes